### ISSUE TITLE
rename package

### ISSUE THREAD

**Issue by beanumber** [SmithCollege-SDS/sds#13]
I think this should be renamed "smith", or "smithsds", but "sds" doesn't mean anything to anyone else. OK with you @rudeboybert @MilesOtt ?

**Comment by rudeboybert** [comment]
I vote for smithsds. may I ask what the concern with recognizability is? are you thinking of publishing on CRAN?

**Comment by beanumber** [comment]
No, but it's already public, so I could imagine other departments might want to use it? I also have some other packages like "sds192" and it's starting to get confusing.
